stdlib/array.gr

```diff
@@ -620,6 +682,9 @@ provide let find = (fn, array) => {
  * @param fn: The function to call on each element, where the returned value indicates if the element satisfies the condition
  * @param array: The array to search
  * @returns `Some(index)` containing the index of the first element found or `None` otherwise
+ * 
+ * @example
+ * Array.findIndex(a => a == 7, [> 5, 6, 7, 8, 9]) // Some(2)
```

Comment:
Same As Above:
```suggestion
 * Array.findIndex(a => a == 7, [> 5, 6, 7, 8, 9]) == Some(2)
```
Might also make sense to add a counter example

Reply:
See comment on .find I would recommend keeping the in line vs. Boolean without an assert or return from the == operation.